src/iexamine.cpp

```diff
@@ -1538,11 +1540,12 @@ void iexamine::fvat_empty(player *p, map *m, int examx, int examy)
         }
         add_msg(_("Set %s in the vat."), brew.tname().c_str());
         m->i_clear(examx, examy);
-        m->i_at(examx, examy).push_back(brew); //This is needed to bypass NOITEM
+        //This is needed to bypass NOITEM
+        m->add_item( examx, examy, brew );
```

Comment:
Except `map::add_item` is too stupid to be used externally. It's, in fact, leading to <http://smf.cataclysmdda.com/index.php?topic=9022.0>. Is there a reason you didn't use `map::add_item_or_charges`?